get-dependencies.js

```diff
@@ -73,7 +73,8 @@ function resolveAllInstalledDependencies(thriftDir, callback) {
             var dir = this.path.slice(0, this.path.length - 1).join('/');
             var includes = parseIncludes(value);
             if (includes.length > 0) {
-                memo[dir] = [].concat(includes.map(pathToServiceName))
+                memo[dir] = (memo[dir] || [])
+                    .concat(includes.map(pathToServiceName))
```

Comment:
Merging includes for a directory makes sense. 👍 

Do you agree that the array should be a set semantically, containing no duplicates? We can de-duplicate the array here, or switch the data-structure.

Reply:
Yeah, this should be a set semantically... wasn't sure if it was worth the complication, when in reality, all duplicate elements cause is a couple extra "file.exists" checks.

Reply:
I think it would be good to de-duplicate. Principle of least surprise.

Reply:
Sounds good, done! Includes are small, so I think it's clean to handle it inline without a data structure change.